src/components/EventMapMarker.css

```diff
@@ -0,0 +1,10 @@
+.Event-dot {
+  width: 12px;
+  height: 12px;
+  background-color: #f243a5;
+  border-radius: 15px;
+  border: 4px solid rgba(255, 255, 255, 0.7);
+  cursor: pointer;
+  -webkit-background-clip: padding-box; /* for Safari */
```

Comment:
You don't need prefixes when you write it in a CSS file in this project. A step on your computer adds prefixes where needed automatically ;)